docs/guides/apache-sites-enabled.md

```diff
@@ -25,27 +25,26 @@ This method uses a couple of additional directories, but they don't currently ex
 
 * From the command-line type `mkdir /etc/httpd/sites-available` and then `mkdir /etc/httpd/sites-enabled`
 
-* We also need a directory where our sites are going to reside. This can be anywhere, but a good way to keep things organized is to create a directory called sub-domains. To keep things simple, put this in /var/www: `mkdir /var/www/sub-domains/`
+* We also need a directory where our sites are going to reside. The directory can be anywhere, but a good way to keep things organized is to create a directory called sub-domains. To keep things simple, put this in /var/www: `mkdir /var/www/sub-domains/`
 
 ## Configuration
 We also need to add a line to the very bottom of the httpd.conf file. To do this, type `vi /etc/httpd/conf/httpd.conf` and go to the bottom of the file and add `Include /etc/httpd/sites-enabled`.
 
 Our actual configuration files will reside in */etc/httpd/sites-available* and we will simply symlink to them in */etc/httpd/sites-enabled*. 
 
 **Why do we do this?**
+The reason here is simple. Say you have ten websites all running on the same server with different IP addresses. Say that site B has some significant updates, and you have to make changes to the configuration for that site. Also, there is something wrong with the changes made, so when you restart httpd to read in the new changes, httpd doesn't start.
 
-The reason here is pretty simple. Let's say you have 10 web sites all running on the same server on different IP addresses. Let's say that site B has some major updates and you have to make changes to the configuration for that site. Let's say too, that there is something wrong with the changes made, so when you restart httpd to read in the new changes, httpd doesn't start. 
+The site you are working on not start, and the other sites also will not. With this method, you can remove the symbolic link for the website that caused the failure and restart httpd. It’ll start working again, and you can go to work, trying to fix the broken site configuration. 
 
```

Comment:
I don't think this sentence reads the way you have intended it to.

Reply:
This was one of two errors I did not know how to correct because I had already pushed the file. I had a similar proble with the TOC review and just submitted another correction after it was merged.

Reply:
You can always push your changes again. In other words. I'm assuming you are creating a branch locally to do your edits. If you do a push for the original PR, simply do that same push again. It should update your PR. It does for me anyway.